src/ViewHoldingsRecord.js

```diff
@@ -352,26 +365,31 @@ class ViewHoldingsRecord extends React.Component {
       />
     );
 
+    const noHoldingsRecordDeleteModalMessageId = itemCount > 1
+      ? 'ui-inventory.itemsOnHoldingsRecordDeleteModal.message'
+      : 'ui-inventory.itemOnHoldingsRecordDeleteModal.message';
+
     const noHoldingsRecordDeleteModalMessage = (
       <SafeHTMLMessage
-        id="ui-inventory.noHoldingsRecordDeleteModal.message"
+        id={noHoldingsRecordDeleteModalMessageId}
         values={{
           hrid: holdingsRecord.hrid,
-          location: (holdingsPermanentLocation ? holdingsPermanentLocation.name : null)
+          location: (holdingsPermanentLocation ? holdingsPermanentLocation.name : null),
+          itemCount,
         }}
       />
     );
 
     const noHoldingsRecordDeleteFooter = (
       <Button onClick={this.hideNoHoldingsRecordDeleteModal}>
-        <FormattedMessage id="stripes-core.button.back" />
+        <FormattedMessage id="stripes-core.button.cancel" />
       </Button>
     );
 
     return (
       <div>
         <ConfirmationModal
-          data-test-delete-confirmation-modal
+          id="data-test-delete-confirmation-modal"
```

Comment:
An `id` of `data-test-...` seems a bit odd. Maybe just `id="delete-confirmation-modal"`?

Reply:
Good catch @zburke! I replaced it with id because `ConfirmationModal` doesn't take the `data-test-*` but I should have changed the name.